bigquery-service/src/main/java/com/github/saintukrainian/bigqueryservice/configs/BigQueryAuthentication.java

```diff
@@ -0,0 +1,54 @@
+package com.github.saintukrainian.bigqueryservice.configs;
+
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.auth.oauth2.ServiceAccountCredentials;
+import com.google.cloud.bigquery.BigQuery;
+import com.google.cloud.bigquery.BigQueryOptions;
+import com.google.cloud.bigquery.Dataset;
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+
+@Slf4j
+public class BigQueryAuthentication {
```

Comment:
Can you check, is it possible to not configure all this stuff manually but have these preconfigured for you by spring?